From linux-usb-devel-bounces@lists.sourceforge.net Wed Aug 15 07:13:31 2007
From: Mike Pagano <mpagano-kernel@mpagano.com>
Date: Wed, 15 Aug 2007 10:13:28 -0400
Subject: USB: resubmission unusual_devs modification for Nikon D80
To: phil@ipom.com, stern@rowland.harvard.edu
Cc: linux-usb-devel@lists.sourceforge.net
Message-ID: <200708151013.28442.mpagano-kernel@mpagano.com>
Content-Disposition: inline


From: Mike Pagano <mpagano-kernel@mpagano.com>

Upgrade the unusual_devs.h file to support the new 1.01 firmware for the Nikon D80.

Signed-off-by: Mike Pagano <mpagano-kernel@mpagano.com>
Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/storage/unusual_devs.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -342,7 +342,7 @@ UNUSUAL_DEV(  0x04b0, 0x040d, 0x0100, 0x
 		US_FL_FIX_CAPACITY),
 
 /* Reported by Emil Larsson <emil@swip.net> */
-UNUSUAL_DEV(  0x04b0, 0x0411, 0x0100, 0x0100,
+UNUSUAL_DEV(  0x04b0, 0x0411, 0x0100, 0x0101,
 		"NIKON",
 		"NIKON DSC D80",
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
